src/FixerFactory.php

```diff
@@ -259,6 +259,7 @@ private function getFixersConflicts(FixerInterface $fixer)
     {
         static $conflictMap = array(
             'no_blank_lines_before_namespace' => array('single_blank_line_before_namespace'),
+            'no_short_echo_tag' => array('opening_tag_plus_echo_to_short_echo_tag'),
```

Comment:
I pondered this for quite some time and I am still not this should be merged into `short_echo_tag` fixer. I understand the fixer as something that changes one aspect of code to one of specific styles in a certain context. Each of these fixers, however, is applied in different context and has a purpose of its own:

`no_echo_tag` fixer is applicable almost everywhere and aims to prevent (most often accidental) use of `<?=`. On the other hand `opening_tag_plus_echo_to_short_echo_tag` will only be useful for templates and is actually meant for harmonization.

I understand, though, that this is somewhat subjective distinction.